docs/_posts/examples/v1.0.0/0100-01-01-leaflet-studio-style.html

```diff
@@ -12,7 +12,8 @@
 <script>
 var map = L.map('map').setView([38.97416, -95.23252], 15);
 
-// Add tiles from Mapbox Style API (https://www.mapbox.com/developers/api/styles/)
+// Add tiles from the Mapbox Static API's retrieve raster tiles from styles endpoint
+// (https://www.mapbox.com/api-documentation/#retrieve-raster-tiles-from-styles)
```

Comment:
I think just calling it the "Mapbox Static API tiles endpoint" might be less confusing/wordy (also in API.md above) and the stylelayer example below